lms/templates/email/instructor_email_digest/body.html.jinja2

```diff
@@ -625,34 +626,50 @@
+                                                                1 student made {{ course.num_annotations }} new annotations in this course.
+                                                              {% else %}
+                                                                A student made a new annotation in this course.
+                                                              {% endif %}
+                                                              {% if course.assignments|length > 0 %}
+                                                                {% for assignment in course.assignments %}
+                                                                  <br/>
+                                                                  <strong>{{ assignment.title }}</strong>
+                                                                  <br/>
+                                                                 {% if assignment.num_annotations > 1 and assignment.annotators|length > 1 %}
+                                                                   {{ assignment.annotators|length }} students made {{ assignment.num_annotations }} new annotations in this assignment.
+                                                                 {% elif assignment.num_annotations > 1 %}
+                                                                   1 student made {{ assignment.num_annotations }} new annotations in this assignment.
+                                                                 {% else %}
+                                                                   A student made a new annotation in this assignment.
+                                                                 {% endif %}
+                                                                {% endfor %}
+                                                              {% endif %}
+                                                            </p>
+                                                          {% endfor %}
 
-                                                            <p><b>We'd love your feedback so we can continue to make improvements
-                                                            to our product. <a href="https://survey.hsforms.com/1pY2LBb_hR9aJMgo9jS0mVQ3quew">Take our survey</a>
-                                                            to tell us what you think of these new daily emails for
-                                                            instructors.</b></p>
+                                                          <p><b>We'd love your feedback so we can continue to make improvements
+                                                          to our product. <a href="https://survey.hsforms.com/1pY2LBb_hR9aJMgo9jS0mVQ3quew">Take our survey</a>
+                                                          to tell us what you think of these new daily emails for
+                                                          instructors.</b></p>
```

Comment:
Various tweaks were made to the wording of the emails, as well as inserting the assignment data. See https://hypothes-is.slack.com/archives/C4K6M7P5E/p1697639042054479